base/strings/util.jl

```diff
@@ -830,7 +830,7 @@ julia> hex2bytes(a)
 """
 function hex2bytes end
 
-hex2bytes(s) = hex2bytes!(Vector{UInt8}(undef, length(s) >> 1), s)
+hex2bytes(s) = hex2bytes!(Vector{UInt8}(undef, length(s)::Int >> 1), s)
```

Comment:
Possibly controversial. However, JuliaHub does not list an InfiniteStrings package (there is an InifiniteArrays package).